docs/schemas/ProductRegistrationEvidenceDocument.json

```diff
@@ -0,0 +1,249 @@
+{
+	"$id": "https://w3id.org/traceability/schemas/ProductRegistrationEvidenceDocument.json",
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$comment": "{\"term\": \"ProductRegistrationEvidenceDocument\", \"@id\": \"https://w3id.org/traceability/ProductRegistrationEvidenceDocument\"}",
+	"title": "Product Registration Evidence Document",
+	"description": "Evidence data for a registered eCommerce Product",
+	"type": "object",
+	"required": [
+		"@context",
+		"category",
+		"inProductGroupWithID",
+		"productID",
+		"mpn",
+		"gtin",
+		"isAccessoryOrSparePartFor",
+		"releaseDate",
+		"manufacturer",
+		"globalLocationNumber",
+		"leiCode",
+		"name",
+		"description",
+		"model",
+		"color",
+		"material",
+		"weight",
+		"height",
+		"width",
+		"depth",
+		"url",
+		"isBasedOn",
+		"image",
+		"HTScode"
+	],
+	"properties": {
+		"@context": {
+			"type": "array"
+		},
+		"type": {
+			"oneOf": [
+				{ "type": "string" },
+				{ "type": "array"}
+			]
+		},
+		"category": {
+			"$comment": "{\"term\": \"category\", \"@id\": \"https://schema.org/category\"}",
+			"title": "A category for the product item.", 
+			"type": "string",
+			"default": "",
+			"examples": [
+				"Example Product Category (required)"
+			]
+		},
+		"inProductGroupWithID": {
+			"$comment": "{\"term\": \"inProductGroupWithID\", \"@id\": \"https://schema.org/inProductGroupWithID\"}",
+			"title": "Indicates the productGroupID for a ProductGroup", 
+			"type": "string",
+			"default": "",
+			"examples": [
+				"valid Product Group ID number per schema.org, could be NULL (required)"
+			]
+		},
+		"productID": {
+			"$comment": "{\"term\": \"productID\", \"@id\": \"https://schema.org/productID\"}",
+			"title": "Product ID", 
+			"type": "string",
+			"default": "",
+			"examples": [
+				"Product number of registering Legal Entity such as an ISBN (required)"
+			]
+		},
+		"mpn": {
+			"$comment": "{\"term\": \"mpn\", \"@id\": \"https://schema.org/mpn\"}",
```

Comment:
today i learned https://schema.org/mpn !

Reply:
http://www.heppnetz.de/projects/goodrelations/
^ It's a product of a very interesting project